js/ui/components/clubhouse.js

```diff
@@ -252,7 +252,7 @@ var ClubhouseComponent = new Lang.Class({
 
         let margin = 50;
         this.actor.x = monitor.width - (this.actor.width + margin);
-        this.actor.y = monitor.height / 2.0 - this.actor.height / 2.0 - margin;
+        this.actor.y = Math.floor((monitor.height - this.actor.height) / 2.0) - margin;
```

Comment:
`ClutterActor.y` is a float. Do we really need this?

Reply:
Yeah, actors need to be grid-aligned, or you'll get artifacts when rendering. In the case of the ASUS E406MA laptop I am testing on, the text was blurry/segmented because the actor was not vertically aligned to the pixel grid.

Reply:
Thanks for the explanation!